french/savoir_africain/poulet/kfc.md

```diff
@@ -0,0 +1 @@
+miam miam te voilà soudain au kfc!! tout t'es offert pour cette fois même les "hots wings".. mais fait gaffe à la malédiction du SURPOIDS sinon tu risques de ne plus être vivant pour résourdre le mystère de ce mur!!
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.